example/src/tests/createdAtTests.ts

```diff
@@ -0,0 +1,548 @@
+import { Test, assert, delayToPropogate, isIos } from './tests'
+import {
+  Client,
+  Conversation,
+  ConversationContainer,
+  Group,
+} from '../../../src/index'
+
+export const createdAtTests: Test[] = []
+
+function test(name: string, perform: () => Promise<boolean>) {
+  createdAtTests.push({ name, run: perform })
+}
+
+test('group createdAt matches listGroups', async () => {
+  // Create three MLS enabled Clients
+  const aliceClient = await Client.createRandom({
+    env: 'local',
+    enableAlphaMls: true,
+  })
+  const bobClient = await Client.createRandom({
+    env: 'local',
+    enableAlphaMls: true,
+  })
+  const camClient = await Client.createRandom({
+    env: 'local',
+    enableAlphaMls: true,
+  })
+
+  // Alice creates a group
+  const aliceGroup = await aliceClient.conversations.newGroup([
+    bobClient.address,
+    camClient.address,
+  ])
+
+  // Bob creates a group
+  const bobGroup = await bobClient.conversations.newGroup([aliceClient.address])
+
+  // Fetch groups using listGroups method
+  await aliceClient.conversations.syncGroups()
+  const aliceGroups = await aliceClient.conversations.listGroups()
+
+  // BUG - List returns in Reverse Chronological order on iOS
+  // and Chronological order on Android
+  const first = isIos() ? 1 : 0
+  const second = isIos() ? 0 : 1
```

Comment:
Do we have a tracking issue for this?

Reply:
https://github.com/xmtp/xmtp-react-native/issues/276, and there's two more issues linked in the description 👍